sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/AbstractInstrument.java

```diff
@@ -88,10 +88,9 @@ public int hashCode() {
 
   abstract static class Builder<B extends AbstractInstrument.Builder<?>>
       implements Instrument.Builder {
-    /* VisibleForTesting */ static final int NAME_MAX_LENGTH = 255;
     /* VisibleForTesting */ static final String ERROR_MESSAGE_INVALID_NAME =
         "Name should be a ASCII string with a length no greater than "
-            + NAME_MAX_LENGTH
+            + StringUtils.NAME_MAX_LENGTH
```

Comment:
while we're here, can we rename this to `METRIC_NAME_MAX_LENGTH` ?

Reply:
Done in 42c17f1